.github/workflows/test-libs-reusable.yml

```diff
@@ -79,7 +78,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref || github.sha }}
-          fetch-depth: 0
+          fetch-depth: 2
```

Comment:
What it the purpose of increasing the `--depth` from 0 to 2 commits from each remote branch ?

Reply:
0 is a special value that means fetch the entire history (for every branch if `ref` not specified)
1 would mean just fetch the state at the latest commit, which is usually what we really want
In this case we want 2 (fetch state at latest commit, and the commit before that too) because [some of the steps in test libs](https://github.com/LedgerHQ/ledger-live/blob/develop/.github/workflows/test-libs-reusable.yml#L96) references the previous commit when doing the checks.

Full docs for `fetch-depth` is [here](https://github.com/actions/checkout?tab=readme-ov-file#usage).

Reply:
clear, thanks